### DISCUSSION TITLE
AlmaLinux not supported?  List of supported distros?

### DISCUSSION THREAD

**Discussion by jebofponderworthy** [searxng/searxng#3271] [Q&A]
When I run the install script on my new Alma Linux 9.3, it seems to indicate that it doesn't know how to do the uwsgi and apache steps on Alma 9.3.  Everything else seems to go through beautifully.  Anything I can do manually to get it working?

**Comment by jebofponderworthy** [comment]
Also appears AlmaLinux 8 is not supported.  Perhaps a list of supported distros might be helpful?  I have had success with Ubuntu 20 and 22 so far.

![image](https://github.com/searxng/searxng/assets/8252044/7631662c-cca6-4fc4-8fc8-f33f31c4a676)

**Reply by unixfox** [reply]
It's a bug. All the Linux distribution should be supported.

**Reply by return42** [reply]
> It's a bug. All the Linux distribution should be supported.

Nope, we support only the major distributions: https://docs.searxng.org/admin/installation-searxng.html#install-packages

Its a lot of work to maintain a distribution, different package manager different distribution releases with different bundled packages ..

**Comment by jebofponderworthy** [comment]
Ditto Rocky 9.
